cmd/serve/unary_handlers.go

```
@@ -0,0 +1,265 @@
+type unaryRequestIDKey struct{}
+
+func unaryRequestID(ctx context.Context) string {
+	v, _ := ctx.Value(unaryRequestIDKey{}).(string)
+	return v
+}
+
+// ---------------------------------------------------------------------------
+// chi handler factories
+// ---------------------------------------------------------------------------
+
+func makeChiCallHandler(p *pool.Pool, methodName string, streamMode bamlutils.StreamMode) http.HandlerFunc {
```

golangci-lint (2.12.1)

[error] func makeChiCallHandler is unused

(unused)